Packs/Exabeam/Integrations/Exabeam/Exabeam.py

```diff
@@ -1078,10 +1118,67 @@ def create_context_table_updates_outputs(name: str, raw_response: Dict) -> Tuple
     return human_readable, entry_context
 
 
+def order_time_as_milisecound_for_fetch(start_time: str, end_time: str) -> Tuple[str, str]:
+
+    date_time = '%Y-%m-%dT%H:%M:%S.%f'
```

Comment:
please rename this TIME_FORMAT_MILISECONDS / TIME_FORMAT (according to what it is) as a constant at the top of the file